enduro.toml

```diff
@@ -83,3 +83,9 @@ disabled = true
 
 [metadata]
 processNameMetadata = false
+
+[worker]
+heartbeatThrottleInterval = "5m"
+
+[activity]
+heartbeatTimeout = "1h"
```

Comment:
After that change, this would be:

```
[workflow]
activityHeartbeatTimeout = "1h"
```

Like we did in other projects.

Reply:
Done.